### PR TITLE
[14.0][FIX] dms: Set res_mimetype correctly in files

### PR DESCRIPTION
**Pull request by victoralmau**
FWP from 13.0: https://github.com/OCA/dms/pull/131

Set `res_mimetype` correctly in files.

Please @Yajo and @pedrobaeza can you review it?

@Tecnativa TT31461

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
If CIs go green

**Comment by victoralmau** [comment]
It's ready to merge right?

File: dms/readme/INSTALL.rst [hunk 1]
```diff
@@ -4,3 +4,6 @@ Preview
 `mail_preview_base` is required for DMS but it is recommended to install all
 the other `mail_preview` modules from `social` OCA repository
 in order to improve the preview of files.
+
+`python-magic` library is recommended to be installed for having whole support
```
Comment (yajo) [inline dms/readme/INSTALL.rst]:
```suggestion
``python-magic`` library is recommended to be installed for having whole support
```

**Review by yajo — CHANGES_REQUESTED** [review]
Just one tiny detail.

**Review by yajo — APPROVED** [review]

**Pull request merged**